pool/paymentmgr.go

```diff
@@ -297,7 +297,7 @@ func (pm *PaymentMgr) PPSSharePercentages() (map[string]*big.Rat, error) {
 // accounts based on work performed measured by the PPLNS payment scheme.
 func (pm *PaymentMgr) PPLNSSharePercentages() (map[string]*big.Rat, error) {
 	now := time.Now()
-	min := now.Add(-(time.Second * time.Duration(pm.cfg.LastNPeriod)))
+	min := now.Add(-pm.cfg.LastNPeriod)
```

Comment:
Could we use `.Sub()` instead of `.Add()` with a negative value here? Same below.

Reply:
Will be maintaining using `Add` here because `Sub` returns the difference (duration) of two time values, `Add` returns a time updated with a duration. 

This is from comments on `Sub`: 
```
...
// To compute t-d for a duration d, use t.Add(-d).
```

Reply:
using `time.Sub` here was ideal https://github.com/decred/dcrpool/pull/151/files#diff-8335d701d364d65ba3e363261132d302R505